foxpuppet/browser/window.py

```diff
@@ -0,0 +1,96 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from .tabbar import TabBar
+from .navbar import Navbar
+from foxpuppet.browser.windows import Windows
+
+NAV_BAR = 'nav-bar'
+TAB_BROWSER = 'tabbrowser-tabs'
+TITLE_BAR_CLOSE = 'titlebar-close'
+TITLE_BAR_MIN = 'titlebar-min'
+TITLE_BAR_MAX = 'titlebar-max'
```

Comment:
These are all IDs, but not all elements will be locatable by ID. I suggest combining the locator with the strategy as a tuple and importing `By` from Selenium. For example:

```python
_nav_bar_locator = (By.ID, 'nav-bar')
_tab_browser_locator = (By.ID, 'tabbrowser-tabs')
```

You can then use `selenium.find_element` like so:

```python
self.selenium.find_element(*self._nav_bar_locator)
```